jme3-desktop/src/main/java/com/jme3/input/awt/AwtMouseInput.java

```diff
@@ -341,4 +347,4 @@ private int getJMEButtonIndex(MouseEvent awtEvt) {
     @Override
     public void setNativeCursor(JmeCursor cursor) {
     }
-}
+}
```

Comment:
One more linebreak here

Reply:
It was probably a couple of accidental keystrokes (I'm working on it).

Reply:
Yeah, it's the opposite, you are missing an empty line here.
That's an arguably odd convention required for linux, because otherwise when inspecting a file in a terminal, the next terminal prompt is written right after the `}` from the source code, so you want to have an empty line, causing linux to start the prompt on the next line.

Reply:
It's strange that this jump doesn't appear in my IDE, but it does appear in a Linux terminal (strange).